### DISCUSSION TITLE
All Products, query returning duplicate in after.

### DISCUSSION THREAD

**Discussion by portedison** [Shopify/storefront-api-feedback#142] [Help]
@johncraigcole  
I'm getting duplicate items when requesting products.

Using 2022-04

```
query getProducts {
  products(first: 4, query: "vendor:chairs* OR product_type:chairs* OR title:chairs* OR tag:chairs*", sortKey: RELEVANCE, after:"[endCursor from previous request]") {
    edges {
      node {
        title
      }
    }
    pageInfo {
      hasNextPage
      endCursor
    }
  }
}

```

The first item is the same item as the last in my previous request (omitting the after).
Potentially only an issue with RELEVANCE, using TITLE doesn't cause an issue.

I think the endCursor is incorrect?

**Comment by johncraigcole** [comment]
Hey @portedison 👋 . Sorry for the slow reply on this. I took a look and I'm seeing the same thing on my end with `RELEVANCE` as the `sortKey`. I'll get an issue created on our end, but won't be able to provide a timeline on when the issue might be resolved.
